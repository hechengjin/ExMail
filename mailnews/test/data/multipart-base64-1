Content-Type: multipart/mixed; boundary=boundary
Content-Transfer-Encoding: base64

This part shouldn't appear
--boundary
Content-Type: text/plain
Content-Transfer-Encoding: base64

TXVsdGlwYXJ0IGJhc2U2NCBlbmNvZGVkIHRleHQu

--boundary--
