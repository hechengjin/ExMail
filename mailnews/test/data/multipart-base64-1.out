Multipart base64 encoded text.